### PR TITLE
Refs #193 - retrieve boot_order

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Makes the boot_order into an attribute and make it available for read.

### CODE REVIEW

File: lib/fog/vsphere/compute.rb [hunk 1]
```diff
@@ -284,6 +287,24 @@ def parent_attribute(path, type)
           nil
         end
 
+        # Maps RbVmomi values to understandable strings.
+        # Result is uniqued, what effectively means that the order is defined by the first occurence.
+        def parse_boot_order(values)
+          return unless values.is_a?(Array)
+          values.map do |v|
```
Comment (timogoebel) [inline lib/fog/vsphere/compute.rb]: nit: can we please call this variable `value`? Thanks.

Comment (ezr-ondrej) [reply lib/fog/vsphere/compute.rb]: I have renamed it to `vm_boot_dev`(s) be more descriptive. Sounds better?

**Comment by ezr-ondrej** [comment]
I forgot small thing :disappointed: but we :green_heart: now

**Comment by timogoebel** [comment]
Thanks, @ezr-ondrej.

**Pull request merged**